test/Helper/FlashMessengerTest.php

```diff
@@ -54,27 +55,28 @@ public function seedCurrentMessages()
 
     public function createServiceManager(array $config = [])
     {
-        return new ServiceManager([
-            'services' => [
-                'Config' => $config,
-            ],
-            'factories' => [
-                'ControllerPluginManager' => function ($services, $name, $options) {
-                    return new PluginManager($services, [
-                        'invokables' => [
-                            'flashmessenger' => 'Zend\Mvc\Controller\Plugin\FlashMessenger',
-                        ],
-                    ]);
-                },
-                'ViewHelperManager' => function ($services, $name, $options) {
-                    return new HelperPluginManager($services, [
-                        'factories' => [
-                            'flashmessenger' => 'Zend\View\Helper\Service\FlashMessengerFactory',
-                        ],
-                    ]);
-                },
-            ],
-        ]);
+        $config = new Config(
+            [
+                'services' => [
+                    'Config' => $config,
```

Comment:
Use lowercase `config` (that's what v2 normalizes to, and ensures forwards compat). (I can do this on merge if needed.)